plugins/vpp/model/acl/keys_agent_acl.go

```diff
@@ -14,7 +14,20 @@
 
 package acl
 
-const aclPrefix = "vpp/config/v1/acl/"
+import "github.com/ligato/vpp-agent/plugins/vpp/model"
+
+const (
+	// DB key prefix
+	aclPrefix = "vpp/config" + model.ProtoApiVersion + "acl/"
+	// REST Acl IP prefix
+	restAclIP = model.ProtoApiVersion + "acl/ip"
```

Comment:
I think it would be probably best to keep REST URI in the REST package. Perhaps we can define constant `acl/` here and use it in REST package.